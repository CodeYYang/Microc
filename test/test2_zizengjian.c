int main(){
    int x=20;
    x++;
    printf("%d",x);
    x--;
    printf("%d",x);
    ++x;
    printf("%d",x);
    --x;
    printf("%d",x);
    int z=2;
    int ans;
    ans=10;
    ans*=z;
    printf("%d",ans);
    ans/=z;
    printf("%d",ans);
    int d=8;
    ans%=d;
    printf("%d",ans);
}