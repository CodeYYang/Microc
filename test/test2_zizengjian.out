24 19 0 5 25 13 0 20 12 13 13 11 0 1 1 12 15 -1 13 11 22 15 -1 13 13 11 0 1 2 12 15 -1 13 11 22 15 -1 13 13 11 0 1 1 12 15 -1 13 11 22 15 -1 13 13 11 0 1 2 12 15 -1 13 11 22 15 -1 13 0 1 1 0 2 12 15 1 13 0 2 1 0 10 12 15 -1 13 0 2 1 13 0 2 1 11 13 0 1 1 11 3 12 15 -1 13 0 2 1 11 22 15 -1 13 0 2 1 13 0 2 1 11 13 0 1 1 11 4 12 15 -1 13 0 2 1 11 22 15 -1 13 0 3 1 0 8 12 13 0 2 1 13 0 2 1 11 13 0 3 1 11 5 12 15 -1 13 0 2 1 11 22 21 4